File: app/src/main/java/com/prey/services/ActivityIntentService.java
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2019 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.services;

import android.content.Intent;
import android.app.IntentService;

import com.prey.PreyConfig;
import com.prey.PreyLogger;

public class ActivityIntentService extends IntentService {

    public ActivityIntentService() {
        super(PreyConfig.TAG);
    }
    @Override
    public void onCreate() {
        super.onCreate();
        PreyLogger.d("ActivityIntentService onCreate");
    }

    @Override
    protected void onHandleIntent(Intent intent) {
    }

}

File: app/src/main/java/com/prey/services/kotlin/ActivityIntentService.kt
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2025 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.services.kotlin

import android.app.IntentService
import android.content.Intent
import com.prey.kotlin.PreyConfig
import com.prey.kotlin.PreyLogger

class ActivityIntentService : IntentService(PreyConfig.TAG) {
    override fun onCreate() {
        super.onCreate()
        PreyLogger.d("ActivityIntentService onCreate")
    }

    override fun onHandleIntent(intent: Intent?) {
    }
}